docs/contribution-guides/3. run-authentication-sample.md

```diff
@@ -0,0 +1,80 @@
+# Contribution Guides
+
+1. **[Get Set up](<./contribution-guides/1. get-set-up.md>)**
+2. **[Build Authentication Server Sample](<./contribution-guides/2. build-authentication-server-sample.md>)**
+3. **[Run Authentication Server Sample](<./contribution-guides/3. run-authentication-server-sample.md>)**
+4. **[Test Your Changes](<./contribution-guides/4. test-your-changes.md>)**
+5. **[Write Unit Tests](<./contribution-guides/5. write-unit-tests.md>)**
+6. **[Submit a PR](<./contribution-guides/6. submit-a-pr.md>)**
+7. **[Publish Your Changes](<./contribution-guides/7. publish-your-changes.md>)**
+
+
+
+## 3. Run Authentication Server Sample
```

Comment:
this is now part of docs/deployment-guides/deploy-locally.md

Reply:
This should not be done by this pr. right?